src/Console/Commands/Install.php

```diff
@@ -9,7 +9,7 @@
  */
 class Install extends GeneratorCommand
 {
-    protected $signature = 'dynamic-forms:install';
+    protected $signature = 'dynamic-forms:install {--local}';
```

Comment:
I do agree that specifying would be better cause if we end up implementing Azure for example then a switch wouldnt work but I didn't like having 'dynamic-forms:install' throw an error because it would just be telling them to include one or the other, I think having a default behavior (--upload=s3) when nothing is included but also making it value based would probably be good

Reply:
Defaulting to S3 is a good idea 👍 

My only concern with a default was the difficulty of swapping if you get it wrong ("oops I didn't realize it defaulted to S3 but I wanted something else") -- but with the rest of the changes being made, it'll probably be as simple as changing the .env setting, so that's not an issue.